<?xml version="1.0" encoding="UTF-8"?>
<launch>
    
    <!--Start Robot Position Controller  -->
    <rosparam file="$(find balance_robot_description)/scripts/control_param.yaml" command="load" ns="/balance_robot_control"/>
    <rosparam file="$(find balance_robot_description)/scripts/robot_param.yaml" command="load" ns="/balance_robot_control"/>
    <node pkg="balance_robot_description" type="balance_robot_control" name="balance_robot_control" 
        output="screen">
    </node>

</launch>